database/redshift/redshift_test.go

```diff
@@ -31,7 +31,7 @@ import (
 var (
 	opts  = dktest.Options{PortRequired: true, ReadyFunc: isReady}
 	specs = []dktesting.ContainerSpec{
-		{ImageName: "postgres:8", Options: opts},
+		{ImageName: "joschi/postgres8:8", Options: opts},
```

Comment:
Why is it needed to pull this image from a non-standard registry location?

Reply:
@shriacquia See https://github.com/golang-migrate/migrate/pull/1128.